src/FlowerShop.Infrastructure/FlowerShop.Infrastructure.csproj

```diff
@@ -18,8 +18,4 @@
   <ItemGroup>
     <ProjectReference Include="..\FlowerShop.Core\FlowerShop.Core.csproj" />
   </ItemGroup>
-  
-  <ItemGroup>
-    <Folder Include="Migrations\" />
```

Comment:
?

Reply:
No clue

Reply:
yeah the csproj likes to do that just leave it as it was before :D